lib/tasks/qunit.rake

```diff
@@ -28,24 +28,14 @@ task "qunit:test", [:timeout, :qunit_path] do |_, args|
     false
   end
 
-  if ENV['QUNIT_EMBER_CLI'] == "0"
-    puts "The 'legacy' ember environment is discontinued - running tests with ember-cli assets..."
-  end
```

Comment:
I think we should keep this message for a few weeks/months, otherwise people are going to be very confused if they look at all the leftover `frontend-legacy` jobs and see they're actually using ember-cli.